src/storage/migration.rs

```diff
@@ -196,7 +196,7 @@ where
             continue;
         };
         let new_key = StorageKey::builder()
-            .start_key(&netflow_raw_event.sensor().unwrap())
+            .start_key(&netflow_raw_event.sensor().expect("always exists"))
```

Comment:
Could you explain why this is safe?

Reply:
- The `sensor` is a method to get the value of the source field of `Netflow5BeforeV23` and `Netflow9BeforeV23`.
- In earlier versions of giganto, before the structure of netflow5 and netflow9 was changed, saving `Netflow5BeforeV23` and `Netflow9BeforeV23` to the DB was done in the following order
  - Receive netflow5, netflow9 from REproduce. (via ingest)
  -   - Extract the source from the certificate in REproduce and store the value in the source field of each structure. 
  - Store the final struct values to the DB.
- Due to the above processing order, the source value always exists, and so do the results of the `sensor` that fetch them.